nsxt/resource_nsxt_policy_ip_block_test.go

```diff
@@ -42,6 +42,40 @@ func TestAccResourceNsxtPolicyIPBlock_minimal(t *testing.T) {
 	})
 }
 
+func TestAccResourceNsxtPolicyIPBlock_v910(t *testing.T) {
```

Comment:
would it make sense to add a test that switches from `cidr` to `cidr_list` with an update?